R/utils.R

```diff
@@ -397,9 +397,11 @@
 #'  between different taxonomic levels, defaults to one compatible with both
 #'  GreenGenes and SILVA `; |;"`.
 #'  
-#' @param column_name a single \code{character} value defining the column of taxa_tab
+#' @param col.name a single \code{character} value defining the column of taxa_tab
 #'  that includes taxonomical information.
 #'  
+#' @param column_name Deprecated. Use \code{col.name} instead.
+#'  
 #' @param remove.prefix {\code{TRUE} or \code{FALSE}: Should 
 #'  taxonomic prefixes be removed? (default: \code{remove.prefix = FALSE})}
```

Comment:
THis is the old name

Reply:
I don't understand.

Reply:
I checked the old version, everything's good